Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
+            string name = PromptString("name");
+            string surname = PromptString("surname");
+            int age = PromptInt("age");
+            int weight = PromptInt("weight");
+            double height = PromptFloat("height");
+            double bmi = CalculateBMI(weight,height);
+
+            Console.WriteLine($"{name} {surname} is {age} years old, his weight is {weight} kg and his height is {height} cm. His BMI is {bmi}.");
+        }
+        public static string PromptString(string attribute)
+        {
+            Console.Write($"What is your {attribute}? ");
+            return Console.ReadLine();
+        }
+        public static int PromptInt(string attribute)
+        {
+            Console.Write($"What is your {attribute}? ");
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static double PromptFloat(string attribute)
+        {
+            Console.Write($"What is your {attribute}? ");
+            return Convert.ToSingle(Console.ReadLine());
+        }
+
+        public static double CalculateBMI(double weight, double height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971681)